### PR TITLE
random

### PR DESCRIPTION
**Pull request by shedaniel**

### CODE REVIEW

File: patchwork-fml/src/main/java/net/minecraftforge/client/ForgeHooksClient.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+
+package net.minecraftforge.client;
+
+public class ForgeHooksClient {
+	public static String forgeStatusLine;
```
Comment (TheGlitch76) [inline patchwork-fml/src/main/java/net/minecraftforge/client/ForgeHooksClient.java]: Might want to add a comment or something here that this isn't actually used anywhere

File: patchwork-fml/src/main/java/net/minecraftforge/fml/BrandingControl.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+
+package net.minecraftforge.fml;
+
+import java.util.List;
+
+public class BrandingControl {
+	private static List<String> brandings;
```
Comment (TheGlitch76) [inline patchwork-fml/src/main/java/net/minecraftforge/fml/BrandingControl.java]: Might want to add a comment or something here that this isn't actually used anywhere

File: patchwork-fml/src/main/java/net/minecraftforge/versions/mcp/MCPVersion.java [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+
+package net.minecraftforge.versions.mcp;
+
+import net.minecraft.SharedConstants;
+
+public class MCPVersion {
+	public static String getMCVersion() {
+		return SharedConstants.getGameVersion().getName();
```
Comment (TheGlitch76) [inline patchwork-fml/src/main/java/net/minecraftforge/versions/mcp/MCPVersion.java]: The indentation looks different for this class than all the others

Comment (shedaniel) [reply patchwork-fml/src/main/java/net/minecraftforge/versions/mcp/MCPVersion.java]: looks the same in ide and they are all tabs, for some reason github is displaying them wrong :hmm:

**Comment by shedaniel** [comment]
plz recheck ty

**Comment by coderbot16** [comment]
:sadcowboy:

**Pull request closed without merging**